plotly/plotly/plotly.py

```diff
@@ -1576,6 +1575,39 @@ def bad_create_animations(fig, kwargs, payload):
     return r_dict
 
 
+def get_uid_by_col_name(grid_url, col_name):
```

Comment:
Also. I sort of imagined that this would have been part of our `Grid` class? More like:

```
1. Get `Grid` from `grid_url` (symmetric to `get_figure`), this is a network operation
2. Get `uid` from `Grid` (which is a simple, local operation)
```